### PR TITLE
Add a command to test block interactions

### PR DESCRIPTION
**Pull request by Karang**
Signed-off-by: Karang arthur.hennequin@free.fr

### CODE REVIEW

**Comment by Spouty** [comment]
**[AUTO-GENERATED COMMENT]**

This pull request has only **_1 commit**_ and is **_signed off**_.
Thanks for following the [pull request guidelines](http://spout.in/prguide) and for your contribution!
A [team member](http://spout.org/team) will review this pull request when time allows.

File: src/main/java/org/spout/engine/command/TestCommands.java [hunk 1]
```diff
@@ -52,6 +57,19 @@ public TestCommands(SpoutEngine engine) {
...
+	public void debugBreak(CommandContext args, CommandSource source) {
+		if (Spout.getPlatform()!=Platform.CLIENT) {
+			
+		}
+		Player player = ((Client) Spout.getEngine()).getActivePlayer();
+		HitBlockComponent hbc = player.get(HitBlockComponent.class);
+		Block block = hbc.getTargetBlock();
+		source.sendMessage("Block to break: ", block.toString());
+		// TODO: A visual to test the code (maybe spawn an entity at the hit location)
+		// (I've tested it with coordinates and it seems to work)
```
Comment (Raphfrk) [inline src/main/java/org/spout/engine/command/TestCommands.java]: Why not just replace the block with air?

Comment (Jckf) [inline src/main/java/org/spout/engine/command/TestCommands.java]: Because that's not what he was testing?

Comment (Raphfrk) [inline src/main/java/org/spout/engine/command/TestCommands.java]: Yes it is.  

The command is debugBreak().  It even says "block to break".  If he also wants to test that drops are spawned, then that is separate, he still needs to clear the target block.

We also need a method which will return the face that is hit.

**Comment by Karang** [comment]
I wasn't sure how to regenerate the chunk after clearing the block so I preferred to let it that way.

**Comment by Raphfrk** [comment]
It should automatically regenerate.  In fact, that would be good to test.

**Pull request closed without merging**